src/XrdPosix/XrdPosixPrepIO.hh

```diff
@@ -48,6 +48,9 @@ long long   FSize() {return (Init() ? fileP->FSize() : openRC);}
 int         Fstat(struct stat &buf)
                  {return (Init() ? fileP->Fstat(buf) : openRC);}
 
+
+virtual int Fcntl(const XrdCl::Buffer& args, XrdCl::Buffer*& res) { return (Init() ? fileP->Fcntl(args, res) : openRC); }
```

Comment:
override, not virtual. probably should do the same to other virtuals of XrdOucIO in here.